imguiWrapper.h

```diff
@@ -7,61 +7,211 @@ extern "C"
 extern void iggWindowSize(IggVec2 *size);
 extern float iggWindowWidth(void);
 extern float iggWindowHeight(void);
+// GetContentRegionMax
 extern void iggContentRegionAvail(IggVec2 *size);
-
+// GetContentRegionAvailWidth
+// GetWindowContentRegionMin
+// GetWindowContentRegionMax
+// GetWindowContentRegionWidth
 extern void iggSetNextWindowPos(IggVec2 const *pos, int cond, IggVec2 const *pivot);
 extern void iggSetNextWindowSize(IggVec2 const *size, int cond);
+// SetNextWindowSizeConstraints
+// SetNextWindowContentSize
+// SetNextWindowCollapsed
 extern void iggSetNextWindowFocus(void);
 extern void iggSetNextWindowBgAlpha(float value);
+// SetWindowPos
+// SetWindowSize
+// SetWindowCollapsed
+// SetWindowFocus
+// SetWindowFontScale
+// SetWindowPos - named
+// SetWindowSize - named
+// SetWindowCollapsed - named
+// SetWindowFocus - named
+
+/*********************/
+/* Windows Scrolling */
+/*********************/
+// GetScrollX
+// GetScrollY
+// GetScrollMaxX
+// GetScrollMaxY
+// SetScrollX
+// SetScrollY
+extern void iggSetScrollHereY(float centerYRatio);
+// SetScrollFromPosY
 
+/******************************/
+/* Parameters stacks (shared) */
+/******************************/
 extern void iggPushFont(IggFont handle);
 extern void iggPopFont(void);
+// PushStyleColor - ImU32
 extern void iggPushStyleColor(int index, IggVec4 const *col);
 extern void iggPopStyleColor(int count);
 extern void iggPushStyleVarFloat(int index, float value);
 extern void iggPushStyleVarVec2(int index, IggVec2 const *value);
 extern void iggPopStyleVar(int count);
-
+// GetStyleColorVec4
+// GetFont
 extern float iggGetFontSize();
-
+// GetFontTexUvWhitePixel
+// GetColorU32
+// GetColorU32Vec4
+// GetColorU32 - ImU32
 extern void iggPushItemWidth(float width);
 extern void iggPopItemWidth(void);
 extern float iggCalcItemWidth(void);
 extern void iggPushTextWrapPos(float wrapPosX);
 extern void iggPopTextWrapPos(void);
+// PushAllowKeyboardFocus
+// PopAllowKeyboardFocus
+// PushButtonRepeat
+// PopButtonRepeat
+
+/*******************/
+/* Cursor / Layout */
+/*******************/
+extern void iggSeparator(void);
+extern void iggSameLine(float posX, float spacingW);
+// NewLine
+extern void iggSpacing(void);
+extern void iggDummy(IggVec2 const *size);
+// Indent
+// Unindent
+extern void iggBeginGroup(void);
+extern void iggEndGroup(void);
+extern IggVec2 iggGetCursorPos(void);
+extern float iggGetCursorPosX(void);
+extern float iggGetCursorPosY(void);
+extern void iggSetCursorPos(IggVec2 const *localPos);
+// SetCursorPosX
+// SetCursorPosY
+extern IggVec2 iggGetCursorStartPos(void);
+extern IggVec2 iggGetCursorScreenPos(void);
+extern void iggSetCursorScreenPos(IggVec2 const *absPos);
+extern void iggAlignTextToFramePadding();
+extern float iggGetTextLineHeight(void);
+extern float iggGetTextLineHeightWithSpacing(void);
+// GetFrameHeight
+// GetFrameHeightWithSpacing
+
+// here because we don't want to break with previous versions of this wrapper
```

Comment:
I don't understand - what is the intent here? These functions are internal, so, they could be removed should they no longer be called from the Go side.